X-Science/StatusWindow.cs

```diff
@@ -323,7 +323,7 @@ public void UpdateSituation( object sender, NewSituationData e )
 			_logger.Trace( "ScienceThisBiome: " + _filter.TotalCount + " / " + _filter.CompleteCount );
 			if( _filter.TotalCount > 0 )
 			{
-				if( _filter.TotalCount - _filter.CompleteCount > 0 )
+				if( _filter.DisplayScienceInstances.Any(x => CanRunExperiment(x)) )
```

Comment:
Can we find a non-LINQ way to do this test? LINQ is great for code readability, but it creates a lot of temporary objects that have to be garbage collected, and that's not good for consistent game performance.